test/srt/test_gguf.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+from pathlib import Path
+
+from huggingface_hub import hf_hub_download
+
+from sglang.srt.configs.model_config import ModelConfig
+from sglang.srt.hf_transformers_utils import get_tokenizer
+from sglang.srt.model_executor.model_runner import ModelRunner
+from sglang.srt.server_args import ServerArgs
+
+
+class TestGGUF(unittest.TestCase):
```

Comment:
This check is too low-level. Please use a more end-to-end test. See also https://github.com/sgl-project/sglang/blob/94e167ea5aaf2871e9176c2b02e4584f5101dd64/test/srt/test_torch_tp.py#L6-L20

Reply:
Is using sgl.Engine good?